### ISSUE TITLE
nixos:release-18.03:nixos.ova.x86_64-linux is blocking the channel

### ISSUE THREAD

**Issue by srhb** [NixOS/nixpkgs#39946]
## Issue description
https://hydra.nixos.org/build/73579774

It appears that the test is unable able to insmod `crc32c-intel`:

```
insmod: can't insert '/nix/store/x7g7dxk6m0di76sbkazwan39hq6vwai0-linux-4.14.39-shrunk/lib/modules/4.14.39/kernel/arch/x86/crypto/crc32c-intel.ko.xz': No such device
```

... promptly followed by a kernel panic.

```
[    0.530488] Kernel panic - not syncing: Attempted to kill init! exitcode=0x00000100
[    0.530488] 
[    0.534044] CPU: 0 PID: 1 Comm: init Not tainted 4.14.39 #1-NixOS
[    0.536397] Hardware name: QEMU Standard PC (i440FX + PIIX, 1996), BIOS rel-1.11.0-0-g63451fca13-prebuilt.qemu-project.org 04/01/2014
[    0.541037] Call Trace:
[    0.542048]  dump_stack+0x5c/0x85
[    0.543370]  panic+0xd5/0x232
[    0.544546]  do_exit+0xad0/0xad0
[    0.545827]  ? handle_mm_fault+0xaa/0x1e0
[    0.547399]  do_group_exit+0x3a/0xa0
[    0.548788]  SyS_exit_group+0x10/0x10
[    0.550237]  do_syscall_64+0x67/0x120
[    0.551673]  entry_SYSCALL_64_after_hwframe+0x3d/0xa2
[    0.553645] RIP: 0033:0x7fc6868780e8
[    0.555053] RSP: 002b:00007ffeabe4bd88 EFLAGS: 00000202 ORIG_RAX: 00000000000000e7
[    0.557964] RAX: ffffffffffffffda RBX: 00000000011a57a0 RCX: 00007fc6868780e8
[    0.560692] RDX: 0000000000000001 RSI: 000000000000003c RDI: 0000000000000001
[    0.563448] RBP: 0000000000000004 R08: 00000000000000e7 R09: ffffffffffffff80
[    0.566201] R10: 0000000000000000 R11: 0000000000000202 R12: 00007ffeabe4c138
[    0.568941] R13: 00007ffeabe4c130 R14: 0000000000000000 R15: 0000000000000000
[    0.572256] Kernel Offset: 0xb600000 from 0xffffffff81000000 (relocation range: 0xffffffff80000000-0xffffffffbfffffff)
[    0.576370] Rebooting in 1 seconds..
Virtual machine didn't produce an exit code.
builder for '/nix/store/pk4czbi5ia0wb99y7vmzr00pcy9api7a-nixos-ova-18.03.132250.3a2ad3e4c03-x86_64-linux.drv' failed with exit code 1
```

### Steps to reproduce
nix-build nixos/release.nix -A ova

## Technical details
release-18.03 3a2ad3e4c03fd30024fafe0f19e3c7e9b7bdc528

**Comment by srhb** [comment]
linux 4.14.38 -> 4.14.39 appears to be the cause:

f92ab67fa51b24e159cf5dc6d770be34582001e5 d9c575be8227b0d9e444575a189a82e21d427ce8

**Comment by dtzWill** [comment]
Good call.  From the changelog[1]:

>       ext4: add MODULE_SOFTDEP to ensure crc32c is included in the initramfs

Not quite sure what to do about that, but sure seems related :).

[1] https://lkml.org/lkml/2018/5/2/648

**Comment by cleverca22** [comment]
this fixes it on release-18.03: #39953
and master: #39954
